x/supplier/client/cli/query_claim_test.go

```diff
@@ -0,0 +1,160 @@
+package cli_test
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	tmcli "github.com/cometbft/cometbft/libs/cli"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/pokt-network/poktroll/testutil/network"
+	"github.com/pokt-network/poktroll/testutil/nullify"
+	"github.com/pokt-network/poktroll/x/supplier/client/cli"
+	"github.com/pokt-network/poktroll/x/supplier/types"
+)
+
+// Prevent strconv unused error
+var _ = strconv.IntSize
```

Comment:
Is this generated or added? Wdyt about including the reason (if we know it) in the comment?

Reply:
Pretty sure this is auto generated @bryanchriswhite